app/components/Login/Login.test.js

```diff
@@ -51,12 +57,14 @@ describe('Login Component Testing', () => {
     expect(wrapper.state().password).toEqual(result2);
   });
 
-  it.skip('should call a function when the form is submitted', () => {
+  it('should call a function when the form is submitted', () => {
     const wrapper = shallow(<Login {...loggedOutProps}/>);
     const loginButton = wrapper.find('.btn-login');
     const loginForm = wrapper.find('.login-form');
 
-    loginForm.simulate('submit');
+    loginForm.simulate('submit', { preventDefault: () => {} });
+
+    expect(handleAuthFn).toHaveBeenCalledTimes(1);
   });
 
   it('should render a welcome message if you are logged in', () => {
```

Comment:
The tests DO represent it, on line 71 of the file. That isn't what changed or have anything to do with the PR, though.